AK/JsonObject.h

```diff
@@ -130,7 +130,7 @@ class JsonObject {
     }
 #endif
 
-    void set(String const& key, JsonValue value)
+    void set(StringView key, JsonValue value)
     {
         m_members.set(key, move(value));
```

Comment:
It's probably better to take a `String` here, `StringView` will force another allocation for `String`s (while a `String` will just ref the `StringImpl`)